textures/metro_control_panel_001_pegboard_wip.py

```diff
@@ -0,0 +1,85 @@
+"""A collection of stuff related to https://stalburg.net/Body_message#Pegs."""
+from typing import Sequence
+
+from infra.utils import chunks
+from textures.metro_control_panel_001_pegboard import (
+    bools_to_int,
+    bools_to_nor_mask,
+    invert_bools,
+    markers_to_bools,
+    metro_control_panel_001_pegboard,
+    PegGroup,
+    pegs_to_bools,
+)
+
+non_print_vals = {*range(33), *range(127, 160), 160, 173}
+
+
+def is_alpha_num(char: str) -> bool:
+    return any(("0" <= char <= "9", "A" <= char <= "Z", "a" <= char <= "z"))
+
+
+alpha_num_ = {*range(48, 58), *range(65, 90), *range(97, 123)}
+
+if __name__ == "__main__":
+
+    # Test applying nor mask to various combinations of pegs
+
+    def color_red(chars: str) -> str:
+        """Colorize alphanumeric characters"""
+        return "".join(f"\x1b[11;31m{char}\x1b[0m" if is_alpha_num(char) else char for char in chars)
```

Comment:
if you want console colors, please add colorama or something like that and don't do custom escape sequences ;)